.circleci/config.yml

```diff
@@ -133,6 +139,31 @@ jobs:
             ./cc-test-reporter upload-coverage
           when: on_success
 
+  e2e_file_upload_features:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /home/circleci/
+      - run:
+          <<: *install_dependencies
+      - run:
+          name: Run e2e file upload features test
+          command: |
+            bash scripts/run_e2e_tests.sh --suite="fileUploadFeatures"
+  
+  e2e_topic_and_story_editory:
```

Comment:
@prasanna08 -- please note typo: editors, not editory.

Reply:
Done.